### DISCUSSION TITLE
Is there Yourtts Model for 44,100hz Sampling Rate?

### DISCUSSION THREAD

**Discussion by chigkim** [coqui-ai/TTS#2539] [General Q&A]
What the title said.
I'm looking for Yourtts model with 44,100hz sampling rate.

**Comment by gunpal5** [comment]
I have same question, but in broad sense. Can we use coqui-ai models in 44,100hz sampling rate?

**Comment by erogol** [comment]
there is no open-source your TTS variant with 44khz since it is hard to find public data to train with. You can use coqui.ai API for it.

**Comment by chigkim** [comment]
Actually VCTK dataset is actually 48kghz with 24bit

I think YourTts used VCTK to train, so they must have down sampled to 16 
before training.
